conda_lock/conda_lock.py

```diff
@@ -372,23 +372,31 @@ def aggregate_lock_specs(lock_specs: List[LockSpecification]) -> LockSpecificati
     return LockSpecification(specs=specs, channels=channels, platform=platform)
 
 
-def _determine_conda_executable(conda_executable: Optional[str], no_mamba: bool):
+def _determine_conda_executable(
+    conda_executable: Optional[str], mamba: bool, micromamba: bool
+):
     if conda_executable:
         if pathlib.Path(conda_executable).exists():
             yield conda_executable
         yield shutil.which(conda_executable)
+
     _conda_exe = ensureconda.ensureconda(
-        mamba=not no_mamba,
-        # micromamba doesn't support --override-channels
-        micromamba=False,
+        mamba=mamba,
```

Comment:
do we have a test that runs a lock using micromamba?

Reply:
no, I will add a test.